committee-code-of-conduct/README.md

```diff
@@ -40,17 +40,17 @@ Our [Incident Report Handling Procedures](incident-process.md) describes how we
 
 The members and their terms are as follows:
 
-### Term ends on August 7, 2021
-
-- Aeva Black (Microsoft)
-- Tasha Drew (VMware)
-
 ### Term ends on August 7, 2022
 
 - Karen Chu (Microsoft)
 - Celeste Horgan (CNCF)
 - Tim Pepper (VMware)
 
+### Term ends on August 23, 2023
+
+- Vallery Lancey (Apple)
+- Nabarun Pal (VMWare)
```

Comment:
```suggestion
- Nabarun Pal (VMware)
```

Reply:
Well, I've been doing this wrong for the past 1.5 years 🤦‍♀️

Reply:
Haha. Folks use both of them interchangeably. The stock ticker (VMW) doesn't help either. 😅